.azure-ci/docker_scripts.sh

```diff
@@ -1,25 +1,33 @@
 #!/bin/bash
-
 set -x
+echo "Start manylinux2010 docker build"
 
 # Upgrading pip and setuptools, TODO: Monitor status of pip versions
 PYTHON_PATH=$(eval find "/opt/python/*${python_ver}*" -print)
-export PATH=${PYTHON_PATH}/bin:${PATH}
+export PATH="${PYTHON_PATH}/bin:${PATH}"
 pip install --upgrade pip==19.3.1 setuptools
 
 # Install CMake
 pip install cmake
 
+# Setup ccache
+yum install -y ccache
+source /io/.azure-ci/setup_ccache.sh
+
+ccache -s
+
 # Install boost
 yum install -y wget tar
 wget https://dl.bintray.com/boostorg/release/1.69.0/source/boost_1_69_0.tar.gz
 tar -zxvf /boost_1_69_0.tar.gz
 mkdir boost
 cd /boost_1_69_0
 ./bootstrap.sh --prefix=/boost
-./b2 install
+./b2 install -j3
 cd ..
 
+ccache -s
```

Comment:
This chows ccache statistics to verify that it is working  (we should see some cache hits)